<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gzonas</class>
 <widget class="QDialog" name="Gzonas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestionar Zona</string>
  </property>
  <widget class="QGroupBox" name="groupBox10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>471</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>Gestionar Zona</string>
   </property>
   <widget class="QTableView" name="lista10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>311</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_10">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>114</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <widget class="QPushButton" name="addbutton10">
       <property name="text">
        <string>Adicionar</string>
       </property>
       <property name="icon">
        <iconset resource="pandasoft.qrc">
         <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="modbutton10">
       <property name="text">
        <string>Modificar</string>
       </property>
       <property name="icon">
        <iconset resource="pandasoft.qrc">
         <normaloff>:/images/document-edit.png</normaloff>:/images/document-edit.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delbutton10">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Eliminar</string>
       </property>
       <property name="icon">
        <iconset resource="pandasoft.qrc">
         <normaloff>:/images/edit-delete.png</normaloff>:/images/edit-delete.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer10">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDockWidget" name="informacion10">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>44</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="features">
        <set>QDockWidget::DockWidgetClosable</set>
       </property>
       <property name="windowTitle">
        <string>Informacion</string>
       </property>
       <widget class="QWidget" name="informacion10_2">
        <widget class="QLabel" name="informacionlabel10">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>111</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="volverboton">
      <property name="toolTip">
       <string>Volver a la Pantalla Principal</string>
      </property>
      <property name="statusTip">
       <string>Volver a la Pantalla Principal</string>
      </property>
      <property name="text">
       <string>Volver</string>
      </property>
      <property name="icon">
       <iconset resource="pandasoft.qrc">
        <normaloff>:/images/go-previous.png</normaloff>:/images/go-previous.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="pandasoft.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>volverboton</sender>
   <signal>clicked()</signal>
   <receiver>Gzonas</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>390</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
